src/test/kotlin/uk/gov/dluhc/printapi/mapper/CertificateMapperTest.kt

```diff
@@ -134,4 +136,21 @@ class CertificateMapperTest {
         verify(printRequestMapper).toPrintRequest(message, ero)
         verify(instantMapper).toInstant(message.applicationReceivedDateTime)
     }
+
+    @Test
+    fun `should addPrintRequestToCertificate that already has a print request`() {
+        // Given
+        val ero = buildEroDto()
+        val message = buildSendApplicationToPrintMessage(certificateLanguage = CertificateLanguageModel.EN)
+        val printRequest = buildPrintRequest()
+        val certificate = buildCertificate(printRequests = listOf(printRequest))
+        val printRequest2 = buildPrintRequest()
+        given(printRequestMapper.toPrintRequest(any(), any())).willReturn(printRequest2)
+
+        // When
+        mapper.addPrintRequestToCertificate(message, ero, certificate)
+
+        // Then
+        assertThat(certificate.printRequests).usingRecursiveComparison().isEqualTo(listOf(printRequest, printRequest2))
```

Comment:
Minor, is it worth doing a verify on printRequestMapper to assert toPrintRequest was called with the correct arguments?